### ISSUE TITLE
Add minSize option

### ISSUE THREAD

**Issue by larionov** [siddharthkp/bundlesize#207]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
bundlesize only checks for `maxSize`

**What is the expected behavior?**
To be able to check `minSize` too.

**If this is a feature request, what is motivation or use case for changing the behavior?**
Would be nice to check for build errors that result in empty or drastically different bundle sizes.

**Comment by siddharthkp** [comment]
🤔 

I don't think this is something bundlesize should do. It can't replace the need for functional tests

**Comment by larionov** [comment]
@siddharthkp I agree that functional tests are needed but just now I had a problem when empty css files was generated in our package. Also, `bundlesize` and `maxSize` names kind of imply that this package checks everything about sizes. I even tried to add `minSize` option in our config without checking docs first.

**Comment by siddharthkp** [comment]
> `bundlesize` and `maxSize` names kind of imply that this package checks everything about sizes

I didn't try to imply that 😅

> I had a problem when empty css files was generated in our package. That requires automated visual tests and that is not fast.

What would you set as the minSize for such case? `> 0`?

**Comment by larionov** [comment]
On Mon, Apr 2, 2018, 19:49 Siddharth Kshetrapal <notifications@github.com>
wrote:

> bundlesize and maxSize names kind of imply that this package checks
> everything about sizes
>
> I didn't try to imply that 😅
>
I can see that 😂

> I had a problem when empty css files was generated in our package. That
> requires automated visual tests and that is not fast.
>
> What would you set as the minSize for such case? > 0?
>
I would set it slightly less then the current size.
And I would say that if the size gets lower intentionally you have to
change maxSize to match that too, to still be able to control it.

**Comment by patrickkunkadazn** [comment]
Would also like this feature. My project outputs a UMD library bundle with other third-party libraries included inside. It would be great for us to be alerted when our bundle size increases or decreases significantly due to a library's size changing.

**Comment by delambo** [comment]
Here's another use case: If you're using webpack with aggressive bundle merging, it's nice to be alerted when a bundle size goes down so that you can reconfigure the `maxSize`.
